### DISCUSSION TITLE
Is there a way of getting the Python code generated from the REPL?

### DISCUSSION THREAD

**Discussion by mmontone** [hylang/hy#2584] [Q&A] [answered]
Hi.
Is there a way of getting the Python code generated when compiling a Hy expression from the REPL?
Let's say I want to see the Python code generated for: `(fn [x] (+ x 1))`

**Comment by Kodiologist** [comment, marked as answer]
Yes, you can use the `--spy` option to `hy`. I'd recommend reading the tutorial for this and many other useful tips.

**Comment by mmontone** [comment]
Also using `hy.disassemble`:
```
(hy.disassemble '(print "Hello World!") True)
```
=> `"print('Hello World!')"`

**Reply by Kodiologist** [reply]
Don't count on it, since it was removed in #2577.

**Reply by mmontone** [reply]
Ah. That's a pity.
